tests/e2e/multigpu/test_eval.py

```diff
@@ -7,13 +7,11 @@
 
 import yaml
 from accelerate.test_utils import execute_subprocess_async
-from tbparse import SummaryReader
 from transformers.testing_utils import get_torch_dist_unique_port
+from utils.tensorboard_ import check_tensorboard
```

Comment:
Why is `utils` sorted as a 3rd party package?

Reply:
ah, conflict in utils under axolotl and utils under tests maybe? let me see if I can make that work properly

Reply:
moving the helper fn and converting to a relative import seems to make it more obvious now.